tests/components/test-components.php

```diff
@@ -431,6 +442,167 @@ public function test_component_post_featured_media() {
 			],
 		];
 
-		$this->assertComponentEquals( $expected, 'irving/post-featured-media' );
+		$component = new Component( 'irving/post-featured-media' );
+
+		$this->assertComponentEquals( $expected, $component );
+	}
+
+	/**
+	 * Test irving/post-list component.
+	 *
+	 * @group core-components
+	 */
+	public function test_component_post_list() {
+		// Demo query args.
+		$query_args = [
+			'post__in' => [ $this->get_post_id() ],
+		];
+
+		// Demo templates.
+		$templates = [
+			'before'  => [
+				[ 'name' => 'example/before' ],
+			],
+			'after'   => [
+				[ 'name' => 'example/after' ],
+			],
+			'wrapper' => [ 'name' => 'example/wrapper' ],
+			'item'    => [ 'name' => 'example/item' ],
+		];
+
+		$expected = $this->get_expected_component(
+			'irving/post-list',
+			[
+				'_alias'   => 'irving/fragment',
+				'children' => [
+					$this->get_expected_component( 'example/before' ),
+					$this->get_expected_component(
+						'example/wrapper',
+						[
+							'children' => [
+								$this->get_expected_component(
+									'irving/post-provider',
+									[
+										'_alias'   => 'irving/fragment',
+										'config'   => [
+											'postId' => $this->get_post_id(),
+										],
+										'children' => [
+											$this->get_expected_component( 'example/item' ),
+										],
+									]
+								),
+							],
+						],
```

Comment:
```suggestion
						]
```

Reply:
Causing travis to fail.

Reply:
Weird. Didn't see this on any of my local tests. Looks like we've got Travis configured to test against PHP versions 7.1 and 7.2. We should probably upgrade those to 7.3 and 7.4 and add a `PHPCompatibilityWP` to our sniffs.